src/main/java/com/elroykanye/userregsystem/rest/UserRegRestController.java

```diff
@@ -38,31 +41,60 @@ public void setUserJpaRep(UserJpaRep userJpaRep) {
 
     @GetMapping("/{user_id}")
     public ResponseEntity<Optional<UserDTO>> getUserById(@PathVariable("user_id") final Long id) {
-        Optional<UserDTO> user = userJpaRep.findById(id);
-        return new ResponseEntity<>(user, HttpStatus.OK);
+        if(userJpaRep.findById(id).isPresent())
+        {
+        	Optional<UserDTO> user = userJpaRep.findById(id);
+            return new ResponseEntity<>(user, HttpStatus.OK);
+        }
+        else
+        {
+        	logger.error("No user found for id : "+id);
```

Comment:
I'm new to using the logger... Haven't even read the docs yet. Where can I see logs that are created by logger objects? In the project dir?

Reply:
You can set, where you want loggers to be printed, in the loggers' XML configuration file by using "appender" tag. One can set loggers to be printed either in console or in separate .log files or maybe even in the database. One can also set the loggers print location in the combination of console, separate log files and database.